crankers/src/bin/main.rs

```diff
@@ -0,0 +1,120 @@
+use std::{collections::HashMap, path::PathBuf, time::Duration};
+
+use anyhow::Context;
+use clap::Parser;
+use jito_bytemuck::AccountDeserialize;
+use jito_vault_core::{vault::Vault, vault_operator_delegation::VaultOperatorDelegation};
+use jito_vault_cranker::{restaking_handler::RestakingHandler, vault_handler::VaultHandler};
+use log::info;
+use solana_rpc_client::nonblocking::rpc_client::RpcClient;
+use solana_sdk::{pubkey::Pubkey, signature::read_keypair_file};
+
+#[derive(Parser)]
+struct Args {
+    /// RPC URL for the cluster
+    #[arg(short, long, default_value = "https://api.devnet.solana.com")]
+    rpc_url: String,
+
+    /// Path to keypair used to pay
+    #[arg(short, long)]
+    keypair: PathBuf,
+
+    /// Vault program ID (Pubkey as base58 string)
+    #[arg(long, default_value = "Vau1t6sLNxnzB7ZDsef8TLbPLfyZMYXH8WTNqUdm9g8")]
```

Comment:
you can read from the client repo as well, which should auto-update with new config/env

Reply:
pretty tricky to convert from Pubkey to OsStr here which clap needs, worth updating? I added `env` so these to make it a bit more flexible if someone needs to update program ids